app-info/generate-eos-extra

```diff
@@ -82,13 +82,27 @@ def main():
                 len(appstream_id.split('.')) > 3):
             appstream_id = appstream_id[:-len(".desktop")]
 
+        # Add XML to include the app in the carousel on the overview page, the
+        # “Editor’s Choice” section on the overview page, the carousel on
+        # category pages, and the “Editor’s Choice” section on category pages.
+        #
+        # This will only work for apps which have a hi-res icon (defined in the
+        # main metainfo entry for that app).
+        #
+        # See https://gitlab.gnome.org/GNOME/gnome-software/-/blob/main/doc/vendor-customisation.md#user-content-featured-apps-and-editors-choice
```

Comment:
Perhaps add an XML example to that page, with comments to clarify which kudo/custom/category causes which behaviours.

Reply:
https://gitlab.gnome.org/GNOME/gnome-software/-/merge_requests/1315